.gitignore

```diff
@@ -1,3 +1,6 @@
-cn*
+cn-*
+cn
```

Comment:
why not keeping `cn*`?

Reply:
because I have a contrib/cn.spec and git consider it as being ignored with cn*